src/app/game-state/inventory.service.ts

```diff
@@ -1012,6 +1012,15 @@ export class InventoryService {
     return itemValue;
   }
 
+  // returns false only for equipment with 0 durability
```

Comment:
```suggestion
  /** returns false only for equipment with 0 durability */
```

Using this format to describe variables is highly recommended as it allows [parsers](https://tsdoc.org/) to easily understand the function.

Developer Use Example in Visual Studio Code: 
![image](https://user-images.githubusercontent.com/3268868/179504453-2eda8c57-1e32-4bf5-bf09-0f6a2e210771.png)